WordPress/Classes/Services/ReaderTopicService.m

```diff
@@ -279,11 +279,10 @@ - (void)followTopicNamed:(NSString *)topicName withSuccess:(void (^)())success f
         }
         return;
     }
-
     ReaderTopicServiceRemote *remoteService = [[ReaderTopicServiceRemote alloc] initWithRemoteApi:[self apiForRequest]];
-    [remoteService followTopicNamed:topicName withSuccess:^{
+    [remoteService followTopicNamed:topicName withSuccess:^(NSNumber *topicID){
         [self fetchReaderMenuWithSuccess:^{
```

Comment:
I think you fixed(?) a different one in [#b86841e](https://github.com/wordpress-mobile/WordPress-iOS/commit/b86841e380103eaac8d36da5fe1be14fad66d0e8). I am not even sure if we even need that one, since `self` is not the one performing the action and I don't think that'd create a retain cycle.

I was talking about [this one](https://github.com/wordpress-mobile/WordPress-iOS/blob/issues/3709-non-ascii-tags/WordPress/Classes/Services/ReaderTopicService.m#L285-L286).